src/components/Buttons.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Buttons({ numbers, onClick }) {
+  return (
+    <p>
+      {
+        numbers.map((number) => (
+          <button
+            type="button"
+            id={number}
+            key={number}
+            onClick={() => {
+              onClick(number);
+            }}
```

Comment:
인라인 함수가 한줄로 표현이 가능하다면 이렇게 해주는게 좋을 것 같네요.
```suggestion
            onClick={() => onClick(number)}
```